app/controllers/projects/samples_controller.rb

```diff
@@ -99,5 +101,26 @@ def current_page
     def set_default_sort
       @q.sorts = 'updated_at desc' if @q.sorts.empty?
     end
+
+    def load_samples
```

Comment:
We had moved this to the application_controller in a previous PR did we not? I believe this method and the three methods below should all go in the `projects/application_controller.rb`. Then `app/controllers/projects/samples/transfers_controller.rb` can be updated to inherit from the `Projects::ApplicationController`

Reply:
I think this is what you meant: 5afa45c